src/main/java/org/cru/godtools/api/translations/GodToolsTranslation.java

```diff
@@ -75,6 +77,19 @@ public static GodToolsTranslation assembleFromComponents(Package gtPackage,
 		return godToolsTranslation;
 	}
 
+	public Optional<PageStructure> getPage(String pageName)
+	{
+		for(PageStructure pageStructure : pageStructureList)
+		{
+			if(Objects.equal(pageStructure.getFilename(), pageName))
+			{
+				return Optional.fromNullable(pageStructure);
```

Comment:
Sure. It could be.  I never remember the difference b/w `of` and `fromNullable` w/o checking, and I didn't in this case.